autoload/inline_edit.vim

```diff
@@ -172,3 +172,84 @@ function! inline_edit#HereDoc()
 
   return [start, end, filetype, indent]
 endfunction
+
+" function! inline_edit#PythonQuotedString() {{{2
+
+function s:check_inside_python_string()
+  return index(map(synstack(line('.'), col('.')), 'synIDattr(v:val, "name")'), "pythonString") >= 0
+endfunction
```

Comment:
Usually, I keep script-local/private functions after the public ones. Could you move this to the bottom of the file? It would also be nice if you could change the under_scored naming style to a CamelCased function.